webdriver/tests/bidi/__init__.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+
+# Compares 2 objects recursively.
+# Actual value can have more keys as part of the forwards-compat design.
+# Expected value can be a callable delegate, asserting the value.
+def recursive_compare(expected: Any, actual: Any) -> None:
+    assert type(expected) == type(actual)
+    if type(expected) is list:
+        assert len(expected) == len(actual)
+        for index, val in enumerate(expected):
+            recursive_compare(expected[index], actual[index])
+        return
+
+    if type(expected) is dict:
+        # Actual dict can have more keys as part of the forwards-compat design.
+        assert expected.keys() <= actual.keys(), \
+            f"Key set should be present: {set(expected.keys()) - set(actual.keys())}"
+        for index, val in enumerate(expected):
```

Comment:
nit: maybe `key` instead of `val` given that we are looping on the keys here?

Reply:
done